config.sh

```diff
@@ -72,9 +82,58 @@ unity_custom() {
     BFOLDER="/system/media/"
     BZIP="bootanimation.zip"
   fi
-}
+  MODTITLE=$(grep_prop name $INSTALLER/module.prop)
+  VER=$(grep_prop version $INSTALLER/module.prop)
+	AUTHOR=$(grep_prop author $INSTALLER/module.prop)
```

Comment:
Is the spacing weird here?

Reply:
i dont see why. its not like that in my editor. ill check again though

Reply:
Oh that's my bad just looked funky